Fuzzlyn/Types/TypeManager.cs

```diff
@@ -100,6 +100,7 @@ public void GenerateTypes()
             SyntaxKind.LongKeyword,
             SyntaxKind.ULongKeyword,
             // string, IntPtr/UIntPtr?
+            // float, double?
```

Comment:
I added support for float/double in https://github.com/jakobbotsch/Fuzzlyn/commit/51b06270331a9783a203237458d9e203d34a0f6f but I disabled it again last month because the JIT does not guarantee bit accurate handling of NaNs between debug and release, and I could not find a way to guarantee that NaNs are never generated.

Reply:
I recently saw that you had disabled it, but I didn't know why.

Reply:
> the JIT does not guarantee bit accurate handling of NaNs between debug and release

Seems like a bug?

Reply:
It's practically impossible to do that because the underlying hardware will canonicalize NaNs as part of various operations. For example, x86 will canonicalize NaNs immediately when it is loaded on the x87 stack, so depending on whether a function is inlined or not this canonicalization will happen.